emf/src/test/resources/properties-instance.json

```diff
@@ -4,6 +4,11 @@
   "nodes": [
     {
       "id": "file_properties-parser-examples-example1-props",
+      "classifier": {
```

Comment:
Is this necessary to change the file? Shouldn't the test pass anyway?

Reply:
The test passes anyway, but I like consistency :)